raiden/network/proxies/token_network.py

```diff
@@ -234,6 +241,12 @@ def new_netting_channel_light(self, creator: Address, partner: Address, signed_t
                     log.critical("new_netting_channel_light failed", **log_details)
                     raise RaidenRecoverableError("creating new channel failed")
             except HTTPError as e:
+                self.proxy.jsonrpc_client.check_for_insufficient_eth(
```

Comment:
first i thought that this must be an invocation to check_why_open_failed, but that function did not exists

we can assume that an HTTPError is probably a balance stuff and check for insufficient eth

this will throw an InsuficientFunds, and if not, it will throw RawTransactionFailed and catched by the is_safe_operation decorator

therefore im ok with this, just toughts